good-enough-practices-for-scientific-computing.tex

```diff
@@ -532,12 +567,14 @@ \subsection*{Acknowledgments}
   contain the project's title, a brief description, up-to-date contact
   information, and
   an example or two of how to run various cleaning or analysis tasks.  It is often
-  the first thing users of your project will look at, so make it
-  explicit that you welcome contributors and point them to ways they
-  can help.
+  the first thing users and collaborators on your project will look at, so 
+ make it explicit how you want people to engage with the project. If you
+ are looking for more contributors, make it
+  explicit that you welcome contributors and point them to the license (more below)
```

Comment:
Why point them to the licence first? Your licence should allow contributions, so I'd rather point them to the CONTRIBUTING file instead...?